packages/edit-site/src/components/post-fields/index.js

```diff
@@ -369,6 +370,7 @@ function usePostFields( viewType ) {
 					return <time>{ getFormattedDate( item.date ) }</time>;
 				},
 			},
+			slugField,
```

Comment:
It's ok for this PR going this way. I'd like to offer some thoughts about next steps.

Initially, the `wordpress/fields` package exported every specific part of a field (edit, label, etc.) and we changed it to the whole field. Now that I see this, I think we have to offer an even higher-level API:

```js
const postFields = usePostFields();
const pageFields = usePageFields();
// etc.
```

Essentially, the [same we do for actions](https://github.com/WordPress/gutenberg/blob/9220e7a0a9f3608743d3ceb029438488a19f4d1e/packages/editor/src/components/post-actions/actions.js#L13). In the future, we'll want 3rd parties to be able to register/unregister fields, so we can't use individual fields in every screen — otherwise, every screen will have to provide a filter/registry for that. Instead, we should offer an API that gives all the fields that are register for a given post type.

Reply:
I've prepared a few small PRs to move the remaining field definition to `@wordpress/fields` in preparation for moving `usePostFields` as well:

- comment_status https://github.com/WordPress/gutenberg/pull/66934
- status https://github.com/WordPress/gutenberg/pull/66937
- date https://github.com/WordPress/gutenberg/pull/66938
- author https://github.com/WordPress/gutenberg/pull/66939
- title https://github.com/WordPress/gutenberg/pull/66940